help/md/fnHKY.md

```diff
@@ -5,20 +5,38 @@ fnHKY
 The Hasegawa-Kishino-Yano (1985) nucleotide rate matrix
 
 ## description
-The HKY85 model.
+DNA evolution model proposed in Hasegawa, Kishino, and Yano (1985).
 
 ## details
+In this model, nucleotides have different stationary frequencies, and transition and transversion rates are allowed to be different. Its first parameter, kappa, codes for the ratio between the rate of transitions and transversions. Its second parameter, baseFrequencies, codes for the frequencies of each nucleotide.
 
-## authors
+The HKY rate matrix elements will be of the form:
+	Q[i, j] = c * kappa * baseFrequencies[j], if i<->j is a transition 
+			= c * baseFrequencies[j], if i<->j is a transversion
```

Comment:
It seems like the two lines here are not horizontally aligned.  Is could this be a spaces/tabs issue?  This also occurs in other files.

Reply:
I'm not sure what to do, on my machine they seem aligned?